vscode-car-plugin/src/main/java/org/wso2/maven/MavenUtils.java

```diff
@@ -88,4 +90,23 @@ public void consumeLine(String line) {
             }
         });
     }
+
+    /**
+     * Checks the project runtime version and decides
+     * whether to pack connector dependencies or not
+     * @param project Maven project
+     * @return whether to pack connector dependencies or not
+     */
+    public static boolean bundleConnectorDependencies(MavenProject project) {
```

Comment:
Calls to this method is always inverted, insted can we invert the logic here and make non-inverted usages?

```
public static boolean bundleConnectorDependencies(MavenProject project) {
        String runtimeVersion = project.getProperties().getProperty(Constants.PROJECT_RUNTIME_VERSION);

        if (runtimeVersion != null) {
            // Compare the version
            ComparableVersion currentVersion = new ComparableVersion(runtimeVersion);
            ComparableVersion targetVersion = new ComparableVersion(Constants.RUNTIME_VERSION);

            return currentVersion.compareTo(targetVersion) < 0;
        }
        return true;
    }
```

Reply:
fixed